Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': SSL connection timeout'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': SSL connection timeout'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Operation timed out
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Operation timed out
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Recv failure: Operation timed out'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Recv failure: Operation timed out'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [rejected]        main -> main (non-fast-forward)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. If you want to integrate the remote changes,
hint: use 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [rejected]        main -> main (non-fast-forward)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. If you want to integrate the remote changes,
hint: use 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Operation timed out
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Operation timed out
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Resolving timed out after 692128 milliseconds'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Resolving timed out after 692128 milliseconds'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Recv failure: Connection reset by peer'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Recv failure: Connection reset by peer'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': SSL connection timeout'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': SSL connection timeout'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Failed to connect to github.com port 443 after 2490239 ms: Timeout was reached'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Failed to connect to github.com port 443 after 2490239 ms: Timeout was reached'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Connection reset by peer
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Connection reset by peer
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Failed to connect to github.com port 443 after 1434836 ms: Timeout was reached'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Failed to connect to github.com port 443 after 1434836 ms: Timeout was reached'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [remote rejected] main -> main (cannot lock ref 'refs/heads/main': is at 2486440ac5a28274d1ca66592b19e41921841ac7 but expected 1fd808c0d275aa94d2a3c4bdb068eea6831df302)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git''
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [remote rejected] main -> main (cannot lock ref 'refs/heads/main': is at 2486440ac5a28274d1ca66592b19e41921841ac7 but expected 1fd808c0d275aa94d2a3c4bdb068eea6831df302)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git''
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [rejected]        main -> main (non-fast-forward)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. If you want to integrate the remote changes,
hint: use 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [rejected]        main -> main (non-fast-forward)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. If you want to integrate the remote changes,
hint: use 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.'
Commit failed: Cmd('git') failed due to: exit code(128)
  cmdline: git add /Users/jacopodalmasso/Desktop/Python/Github/autocommit/counter.txt /Users/jacopodalmasso/Desktop/Python/Github/autocommit/commit_log.txt /Users/jacopodalmasso/Desktop/Python/Github/autocommit/error_log.txt
  stderr: 'fatal: Unable to create '/Users/jacopodalmasso/Desktop/Python/Github/autocommit/.git/index.lock': File exists.

Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue.'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git add /Users/jacopodalmasso/Desktop/Python/Github/autocommit/counter.txt /Users/jacopodalmasso/Desktop/Python/Github/autocommit/commit_log.txt /Users/jacopodalmasso/Desktop/Python/Github/autocommit/error_log.txt
  stderr: 'fatal: Unable to create '/Users/jacopodalmasso/Desktop/Python/Github/autocommit/.git/index.lock': File exists.

Another git process seems to be running in this repository, e.g.
an editor opened by 'git commit'. Please make sure all processes
are terminated then try again. If it still fails, a git process
may have crashed in this repository earlier:
remove the file manually to continue.'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [remote rejected] main -> main (cannot lock ref 'refs/heads/main': is at 623e5680fd3b1abb78d4d028839d08e033e313e1 but expected 15783491da2d0943aaa48f41ebc0d746ddb156a4)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git''
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [remote rejected] main -> main (cannot lock ref 'refs/heads/main': is at 623e5680fd3b1abb78d4d028839d08e033e313e1 but expected 15783491da2d0943aaa48f41ebc0d746ddb156a4)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git''
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [remote rejected] main -> main (cannot lock ref 'refs/heads/main': is at a3c145a70973803bb756aa3b5db7d999ce8f49c8 but expected 270354c3651fbf0b816f5c0d35075c0015630f98)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git''
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'To https://github.com/jdalmasso/autocommit.git
 ! [remote rejected] main -> main (cannot lock ref 'refs/heads/main': is at a3c145a70973803bb756aa3b5db7d999ce8f49c8 but expected 270354c3651fbf0b816f5c0d35075c0015630f98)
error: failed to push some refs to 'https://github.com/jdalmasso/autocommit.git''
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Recv failure: Connection reset by peer'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Recv failure: Connection reset by peer'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Connection reset by peer
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Connection reset by peer
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Push failed: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Connection reset by peer
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Script error: Cmd('git') failed due to: exit code(1)
  cmdline: git push
  stderr: 'error: RPC failed; curl 56 Recv failure: Connection reset by peer
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date'
Push failed: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
Script error: Cmd('git') failed due to: exit code(128)
  cmdline: git push
  stderr: 'fatal: unable to access 'https://github.com/jdalmasso/autocommit.git/': Could not resolve host: github.com'
